### PR TITLE
[RL][00000] Update readme

### PR DESCRIPTION
**Pull request by rkledesma**
Update with instructions for MacOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Usage](#usage)
+- [Installation](#installation)
+  - [MacOS](#macos)
+  - [Other OS](#other-os)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```
Comment (colinrymer) [inline README.md]: Can we move the TOC to below the title and description (what is now lines 12-14)?

Comment (colinrymer) [inline README.md]: i.e. like on https://github.com/thlorenz/doctoc

File: README.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Usage
+
+- [Usage](#usage)
+- [Installation](#installation)
+  - [macOS](#macos)
+  - [Other OS](#other-os)
 
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 ## Usage
```
Comment (colinrymer) [inline README.md]: The Usage header is now duplicated.

File: README.md (hunk 1 above)
Comment (rkledesma) [inline README.md]: Sure; I've made the change.

**Comment by colinrymer** [comment]
@rkledesma If you'll fix the duplicate `## Usage` header and squash your commits, this looks like it'll be good to go. Thanks for adding the ToC! :+1:

File: README.md [hunk 3]
```diff
@@ -17,9 +25,12 @@ $ rpenv <env> <cmd>
 
 where `<env>` is one of `ci`, `qa`, or `prod` (`production` should also work) and `<cmd>` is the desired command you wish to run. If called without a `<cmd>`, `rpenv` will return a list of all the env vars in the `/etc/rentpath/environment.cfg` file merged with your current environment variables (i.e. whatever `/usr/bin/env` would return). When run with a `<cmd>`, it will execute that `<cmd>` after setting the environment with the values returned if `rpenv` is run without a `<cmd>`.
 
-Build
-====
+## Installation
 
+### macOS
+brew update && brew tap rentpath/homebrew && brew install rentpath/homebrew/rpenv
```
Comment (pasha) [inline README.md]: this reads like a text, please ether wrap it with ` or add 4 spaces

**Comment by pasha** [comment]
👍

**Comment by colinrymer** [comment]
:+1: LGTM Be sure to squash before merging.

**Pull request merged**